### DISCUSSION TITLE
What is criteria to download the User profile exhaust' report?

### DISCUSSION THREAD

**Discussion by vishalchincholi1** [Sunbird-Obsrv/Community#99] [Query]
Hello All,
I am trying to download the User profile exhaust' report from the view course dashboard. After submitting the request the status is getting displayed as failed. 
I have checked that the jobs are working fine from the backend, but I am unable to understand what is criteria to download the user profile exhaust report. 
![user exhaustive report](https://github.com/Sunbird-Obsrv/Community/assets/124983642/d40ff9fc-e9fe-4f2c-89b2-e784ccbe545d)
Can anyone please guide me here ?
CC @komalm @Renuka-S @sravani-64 @Nithin-ns24

**Comment by manjudr** [comment]
@vishalchincholi1  - Kindly check the Postgres job request table, where you will find both the jobs status as well as the reason for failure if the job has encountered any issues

**Reply by manjudr** [reply]
@vishalchincholi1  - Any updates on this issue?

**Reply by vishalchincholi1** [reply]
Not yet

**Reply by manjudr** [reply]
@vishalchincholi1   Ok - If any of the report requests are failed then it will be marked as failed requests along with the detailed error message.  If you have access, Either you can you the report read API to pull the request status or you can manually check in the Postgres job_request table.
